src/main/groovy/edu/oregonstate/mist/idimages/resources/IdImagesResource.groovy

```diff
@@ -35,21 +37,46 @@ class IdImagesResource extends Resource {
     @Produces("image/jpg")
     @Timed
     public Response getByOSUID(@Auth AuthenticatedUser _, @PathParam('id') String id) {
+        Integer bannerPIDM = idImageDAO.getPIDM(id)
+
+        if (!bannerPIDM) {
+            return notFound().type(MediaType.APPLICATION_JSON).build()
+        }
         Blob image = idImageDAO.getByID(id)
 
+        //Return a placeholder image if a person exists but doesn't have an ID image
         if (!image) {
-            return notFound().type(MediaType.APPLICATION_JSON).build()
+            return ok(getImageStream(ImageIO.read(new File("images/defaultImage.jpg")))).build()
```

Comment:
Where did we get the placeholder image? I think we should get their permission to use the default image.